pages/linux/fossil-add.md

```diff
@@ -0,0 +1,12 @@
+# fossil add
+
+> Put files or directories under Fossil version control.
```

Comment:
> I'm not sure if that's correct. If you change a file which already is under version control, you still have to add it, right?

No, that's not the case in Fossil.  ie, Fossil does not have the Git concept of the index/staging-area.  When you commit in Fossil, all pending changes are committed.  Thus, add and remove are solely concerned with what is or is not tracked under version control (though with remove you can also optionally also actually remove the file from the local checkout).

That said, if my verbiage is confusing to you, it will likely be so to others.  Given my comments above, would you still like to me to proceed with your suggested change?  I have no objections...  Thoughts @marchersimon ?

Reply:
In this case your wording is perfectly fine. I just thought it was more similar to git.